tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/MainActivity.kt

```diff
@@ -20,10 +21,9 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val ditto = TasksApplication.ditto
         try {
-            ditto!!.setOfflineOnlyLicenseToken("<REPLACE_ME>")
-            ditto!!.startSync()
+//            ditto?.setOfflineOnlyLicenseToken("<REPLACE_ME>")
```

Comment:
If this app is only going to use `OnlinePlayground` then this is dead code.

Reply:
⚰️removed